testscpp/tests/test_presolve.cpp

```diff
@@ -113,8 +113,8 @@ SCENARIO("constrained quadratic models can be presolved") {
                 }
 
                 AND_WHEN("we then undo the transformation") {
-                    auto original = presolver.restore(std::vector<int>{3});
-                    CHECK(original == std::vector<int>{3, 5, 7, 5});
+                    auto original = presolver.restore(std::vector<double>{3});
```

Comment:
These have to be `double`s now because you have to declare the templated classes explicitly and you only have `double` for `Assignment` currently?

Reply:
Yes, exactly. It also makes more sense this way. If we're going to bother specifying the `assignment_type` at all, it's weird that we would suddenly not be picky about that here.